csharp/ql/lib/semmle/code/csharp/dataflow/internal/ContentDataFlow.qll

```diff
@@ -0,0 +1,432 @@
+/**
+ * Provides classes for performing global (inter-procedural)
+ * content-sensitive data flow analyses.
+ */
+
+private import DataFlowImplCommon
+private import DataFlowPrivate as DataFlowPrivate
```

Comment:
Isn't this included in `private import DataFlowImplSpecific::Private` ?

Reply:
Ah yes, it is wrong to rely on the existence of a `DataFlowPrivate.qll` module.